### ISSUE TITLE
React Native ?

### ISSUE THREAD

**Issue by yoongfook** [mgcrea/redux-rest-resource#10]
First of all, i love this library so much. It makes me so productive. 

I am just wondering how difficult is it to make it work for React Native? I know you are busy, but I am wondering if it is something easy i can work on, or is it hard?

I am getting the error below:
<img width="487" alt="screen shot 2017-05-04 at 9 00 11 pm" src="https://cloud.githubusercontent.com/assets/1704381/25700823/bb228882-310c-11e7-8b03-0a29964e85cd.png">

**Comment by peritus** [comment]
@yoongfook here is what I added to my `package.json` to make this library work in react-native:

```diff
diff --git a/package.json b/package.json
index 55327793..2cc9d449 100644
--- a/package.json
+++ b/package.json
@@ -18,7 +18,8 @@
+    "postinstall": "rm ./node_modules/redux-rest-resource/.babelrc; sed -iorig 's/self/global/g' node_modules/isomorphic-fetch/fetch-npm-browserify.js"
   },
   "dependencies": {
     "bluebird": "^3.5.0",
```

@mgcrea can we include this in a future release ?

Unsure about the second command (patching `isomorphic-fetch`). React Native ships with a working `fetch` implementation, so it's not needed (or should detect that it's already there).

**Comment by mgcrea** [comment]
@peritus since we merged #13, does it works out of the box with react-native?

**Comment by peritus** [comment]
@mgcrea Yes, just now removed the hack I mentioned above from our code base and it works fine. I guess this can be closed then. Thanks.
